nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+
```

Comment:
Here is where you should create the different routes, like for example: 

/meals
If we call any of these routes, it should return a list of meals.